### DISCUSSION TITLE
Лайкер. Порядок добавления треков.

### DISCUSSION THREAD

**Discussion by acidspell** [Chimildic/YaMuTools#44] [Идея] [answered]
Доброго времени суток, на данный момент лайкер лайкает треки сверху вниз, из-за этого треки переносятся в "мне нравится" в обратном порядке, так как там всегда новые треки сверху. Предлагаю сделать работу лайкера снизу вверх.

**Comment by Chimildic** [comment, marked as answer]
Лайкер добавляет треки в обратном порядке с версии 0.8.0. Также появилась сортировка-реверс для обычных плейлистов.
